src/lib/utils/options/readers/tsconfig.ts

```diff
@@ -59,8 +59,14 @@ export class TSConfigReader extends OptionsComponent {
                 return;
             }
         } else if (TSConfigReader.PROJECT_KEY in event.data) {
-            // The `project` option may be a directory or file, so use TS to find it
-            file = ts.findConfigFile(event.data[TSConfigReader.PROJECT_KEY], ts.sys.fileExists);
+            const resolved = Path.resolve(event.data[TSConfigReader.PROJECT_KEY]);
+            // If the file exists, use it
+            if (FS.existsSync(resolved)) {
```

Comment:
Does this need to check if it is a file too? What doesn't `ts.findConfigFile` do correctly?

Reply:
Yes, this should also check for a file, `ts.findConfigFile` searches for `tsconfig.json` (or other file passed in to the function - 3rd argument) and doesn't check if the first argument is a file.